credentials/apps/edx_credentials_extensions/edly_credentials_app/api/v1/views/edly_sites.py

```diff
@@ -72,7 +72,7 @@ def process_client_sites_setup(self):
                 edx_org_short_name=edly_slug,
                 platform_name=self.request.data.get('platform_name', ''),
                 company_name=self.request.data.get('platform_name', ''),
-                theme_name=theme_dir_name,
+                theme_name=theme_dir_name if theme_dir_name in AVAILABLE_THEMES else DEFAULT_THEME,
```

Comment:
@abdulmanann since we don't support theming in credentials and don't intend to add it any time soon, we can simply assign the `DEFAULT_THEME` var here.

Reply:
I noticed this [ticket](https://edlyio.atlassian.net/browse/EDLY-6658) in the backlog for designing a program certificate template, which is why I included the `AVAILABLE_THEMES` option. However, if preferred, I can remove it.

Reply:
This is for adding custom certificate templates, not related to our theming